src/components/redux/reducers/index.jsx

```diff
@@ -1,16 +1,65 @@
 import { combineReducers } from "redux";
+import {
+  ADD_TO_CART,
+  DELETE_FROM_CART,
+  SUM_TOTAL_PRICE,
+  ADD_ONE_MORE_ITEM,
+  REMOVE_ITEM,
+  QUANTITY_IN_SINGLE_PRODUCT,
+  INCREASE_MODIFY_QUANTITY,
+  DECREASE_MODIFY_QUANTITY,
+} from "../actions/actionsTypes";
 
-const currentBoardIDReducer = (state = [], action) => {
+const addOrDeleteProductData = (state = [], action) => {
   switch (action.type) {
-    case "CURRENT_BOARD_ID":
+    case ADD_TO_CART:
+      return [...state, action.payload];
+    case DELETE_FROM_CART:
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+const getModifiedQuantity = (state = [], action) => {
+  switch (action.type) {
+    case INCREASE_MODIFY_QUANTITY:
+      return action.payload();
+    case DECREASE_MODIFY_QUANTITY:
+      return action.payload();
+    default:
+      return state;
+  }
+};
+
+// Getting an array of prices , and enabling adding and removing items from the shopping cart.
+const getTotalPrice = (state = [], action) => {
+  switch (action.type) {
+    case SUM_TOTAL_PRICE:
+      return [...state, action.payload];
+    case ADD_ONE_MORE_ITEM:
+      return [action.payload];
+    case REMOVE_ITEM:
+      return [action.payload];
+    default:
+      return state;
+  }
+};
+
+const getProductQunatity = (state = [], action) => {
+  switch (action.type) {
+    case QUANTITY_IN_SINGLE_PRODUCT:
       return action.payload;
     default:
       return state;
   }
 };
 
 const allReducers = combineReducers({
-  currentBoardIDReducer,
+  addOrDeleteProductData,
```

Comment:
see my comment on emre's pr or talk to emre, usually these should not be function names, they should be domains.

so for example
```
state.productQuantities
state.prices
state.productData
```